packages/blade/src/tokens/global/typography.ts

```diff
@@ -0,0 +1,101 @@
+import type { FontFamily } from './fontFamily';
+import fontFamily from './fontFamily';
+
+type FontWeight = {
+  regular: 400;
+  bold: 700;
+};
+
+type FontSize = {
+  10: number;
+  25: number;
+  50: number;
+  75: number;
+  100: number;
+  200: number;
+  300: number;
+  400: number;
+  500: number;
+  600: number;
+  700: number;
+  800: number;
+  900: number;
+  1000: number;
+};
+
+const fontWeight: FontWeight = {
+  regular: 400,
+  bold: 700,
+};
+
+export type Typography = Record<
+  'desktop' | 'mobile',
+  {
+    fonts: {
+      family: FontFamily;
+      size: FontSize;
+      weight: FontWeight;
+    };
+  }
+>;
+
+const typography: Typography = {
+  mobile: {
+    fonts: {
+      family: {
+        ...fontFamily,
+      },
+      size: {
+        10: 10,
+        25: 11,
```

Comment:
Why are we having a separate key to value mapping? Why not same value in key and value?
```
12: 12,
14: 14,
```